scripts/prepare_release.sh

```diff
@@ -1,5 +1,16 @@
 #!/bin/bash
-#
+
+print_usage()
+{
+    cat <<EOF
+Usage: $0 [OPTION]...
+Prepare the source tree for a release.
+
+Options:
+  -u    Prepare for development (undo the release preparation)
```

Comment:
Do we really need an “unrelease” mode? I kept it from `gitignore_patch.sh`, but I'm not sure it's actually useful. We normally revert the release preparation commit(s) to go from release mode to development mode. Having this mode lets us do a bug fix on the release branch and get it back into development without merging, but we don't really need the “without merging” property?

Reply:
That would be outside of the scope of this PR/ASk though. I cannot speak for others since I have not used this particular script, but if people are using it it should stay. This is a helper tool for the maintainers anyway. We can always revisit this.